RELEASE_NOTES.md

```diff
@@ -2,6 +2,29 @@
 
 This page describes the noteworthy improvements provided by each release of Eclipse Tycho.
 
+## 4.0.3
```

Comment:
just in case it is sufficient for a backport to only have a short summary of what was done  but the "long form" is fine as well)

Reply:
I was wondering about that point because actually this feature is then not new in 5 because it will be first introduced in 4.0.3.
But since they are on different branches it will not appear on master with 4.0.3.
Nevertheless it could be confusing if one has used a feature and now reads that it is new in a later version.
Do you plan to 'forward' port the Release notes on a 4.0.3 or at least 5 release to the master?

Reply:
Usually I copy them after the realese from the release branch to the master.